### PR TITLE
add two selects to the top of the label and api selectmodule and add container to the selects items  and give some sytle to the container

### PR DESCRIPTION
**Pull request by yosefmare**
add two selects to the top of the label
the first one for the Date Range
and the second one for the Account
add container to the selects items 
and give some sytle to the container

### CODE REVIEW

File: src/app/shared/data-table/data-table.component.html [hunk 1]
```diff
@@ -2,6 +2,33 @@
 	<div class="data-table-header">
 		<button mat-raised-button class="data-table-add-button" color="primary" (click)="addButtonClicked()">Add</button>
 	</div>
+
+
+<div class="selects-container">
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: They should be part of the header `data-table-heade`

File: src/app/shared/data-table/data-table.component.html [hunk 2]
```diff
@@ -2,6 +2,33 @@
...
+  <div class="select-date">
+    <mat-form-field appearance="fill">
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: Please change the appearance to standard

File: src/app/shared/data-table/data-table.component.html [hunk 3]
```diff
@@ -2,6 +2,33 @@
...
+  <mat-select>
+    <mat-option *ngFor="let Date of ranges" [value]="Date.value">
+      {{Date.viewValue}}
+    </mat-option>
+  </mat-select>
+</mat-form-field>
+  </div>
+
+  <div class="select-Account">
+    <mat-form-field appearance="fill">
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: Please change the appearance to standard

File: src/app/shared/data-table/data-table.component.html [hunk 4]
```diff
@@ -2,6 +2,33 @@
...
+  <mat-label>Date Range</mat-label>
+  <mat-select>
+    <mat-option *ngFor="let Date of ranges" [value]="Date.value">
+      {{Date.viewValue}}
+    </mat-option>
+  </mat-select>
+</mat-form-field>
+  </div>
+
+  <div class="select-Account">
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: please use only lowercase and snake-case for classes

File: src/app/shared/data-table/data-table.component.html [hunk 5]
```diff
@@ -2,6 +2,33 @@
...
 		<button mat-raised-button class="data-table-add-button" color="primary" (click)="addButtonClicked()">Add</button>
 	</div>
+
+
+<div class="selects-container">
+  <div class="select-date">
+    <mat-form-field appearance="fill">
+  <mat-label>Date Range</mat-label>
+  <mat-select>
+    <mat-option *ngFor="let Date of ranges" [value]="Date.value">
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: Please change Dato to use lower case

File: src/app/shared/data-table/data-table.component.html [hunk 6]
```diff
@@ -2,6 +2,33 @@
...
+    </mat-option>
+  </mat-select>
+</mat-form-field>
+  </div>
+
+  <div class="select-Account">
+    <mat-form-field appearance="fill">
+      <mat-label>Account</mat-label>
+      <mat-select>
+        <mat-option *ngFor="let Account of Accounts" [value]="Account.value">
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.html]: Please change account to use lowercase

File: src/app/shared/data-table/data-table.component.ts [hunk 7]
```diff
@@ -50,4 +50,18 @@ export class DataTableComponent<T> implements OnInit {
...
+
+
+  ranges = [
+    {value: 'This Month', viewValue: 'This Month'},
+    {value: 'Previous Mons', viewValue: 'Previous Mons'},
+    {value: '2 Month Ago', viewValue: '2 Month Ago'},
+    {value: '3 Month Ago', viewValue: '3 Month Ago'},
+  ];
+
+  Accounts = [
```
Comment (EzDor) [inline src/app/shared/data-table/data-table.component.ts]: Please use camelCase for variables.
Please add public and private to class members and locate them in the top of the class

File: src/app/shared/data-table/data-table.component.html (hunk 1 above)
Comment (yosefmare) [reply src/app/shared/data-table/data-table.component.html]: fixed

**Review by EzDor — APPROVED** [review]

**Pull request merged**